Mod Manager/src/routes/modList.svelte

```diff
@@ -34,6 +34,29 @@
 
 	$: disabledMods = getAllMods()
 		.filter((a) => !getConfig().loadOrder.includes(a))
+		.sort((a, b) => {
+			function getNonFrameworkModName(id: string) {
+				let parts = id.split('.')
```

Comment:
What's the purpose of this? As far as I can tell, this function is used for getting the names of RPKG (non-framework) mods, but the names of RPKG mods are user-entered and appear in the load order by those names.

Reply:
It's because I at first always used the load order name so I had to remove the author from the id and didn't realise that rpkg mod ids are not built like that.